### PR TITLE
feat: Create UI empty state for customer table

### PR DESCRIPTION
**Pull request by Lanng21**
# Description

- Create UI empty state for customer table
image

## Detailed PR Verification Procedure(s)

- Go to `/components` to check the UI for customer table

## Ticket

[GS-33](https://tito-training.atlassian.net/browse/GS-33)

## Application Checklist

- [x] **JIRA Linked**: Author has followed template and has a title in the following format:
  > GS-33: Create UI empty state for customer table
- [ ] **No Breaking Changes**: Author has confirmed all API contract changes are non-breaking.
- [ ] **Changes Unit Tested**: Author has unit tested new code to prove it works.
- [ ] **Application Passes All Tests**: Author ensures this pull-request passes `npm test`.
- [x] **Author Self Reviewed**: Author of pull-request has self-reviewed this code.
- [x] **Author is Assignee**: Author has assigned self as assignee of pull request.

## Security Checklist

- [ ] **Code Review for Security:** Developer has self-reviewed this code, based on OWASP Top-10 vulnerabilities and general environment security attributes, to avoid dversely impact the security of the system/environment/application, to the best of their abilities.
- [ ] **Backout Procedures:** Developer has considered how the changes presented in this pull request can be replaced with a prior version of these changes, fixed going forward, and/or implemented to be backwards compatible.

### CODE REVIEW

**Comment by baonguyen26** [comment]
With `Detailed PR Verification Procedure(s)` it should 
- Go to `/components` to check the UI for customer table
**please know what you're doing, don't just copy**

File: src/views/components/common/Empty/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import EmptyCustom from './styles'
+
+interface Iempty {
```
Comment (baonguyen26) [inline src/views/components/common/Empty/index.tsx]: IEmpty

File: src/views/components/index.tsx [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+    console.log('import')
+  }
+  const handleAddCustomer = () => {
+    console.log('add')
+  }
+  return (
+    <div>
+      <Empty
+        emptyImg={emptyImg}
+        content='Quản lý danh sách khách hàng một cách dễ dàng và tiện lợi'
```
Comment (baonguyen26) [inline src/views/components/index.tsx]: content should be props

Comment (baonguyen26) [reply src/views/components/index.tsx]: do not fixed it with a content

File: src/views/components/index.tsx [hunk 3]
```diff
@@ -1,7 +1,34 @@
 import React from 'react'
```
Comment (baonguyen26) [inline src/views/components/index.tsx:1]: Why is it a file in the components folder?

File: src/views/components/index.tsx [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+import emptyImg from '@/assets/icons/Client.svg'
+import Empty from './common/Empty'
+import { ReactComponent as AddCustomer } from '@/assets/icons/AddCustomer.svg'
+import { ReactComponent as FileImport } from '@/assets/icons/FileImport.svg'
+import Button from './common/Button'
 
 const Components = () => {
-  return <div>Test components in here</div>
+  const handleImpot = () => {
+    console.log('import')
```
Comment (baonguyen26) [inline src/views/components/index.tsx]: TODO

File: src/views/components/index.tsx [hunk 5]
```diff
@@ -1,7 +1,34 @@
...
+  }
+  const handleAddCustomer = () => {
+    console.log('add')
```
Comment (baonguyen26) [inline src/views/components/index.tsx]: TODO

File: src/constants/Customer.ts [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const Customer = {
+  desEmptyCustomer: 'Quản lý danh sách khách hàng một cách dễ dàng và tiện lợi',
```
Comment (haunguyen15-tt) [inline src/constants/Customer.ts]: translation

**Review by baonguyen26 — APPROVED** [review]

**Pull request merged**